React/Views/RCTScrollView.m

```diff
@@ -352,7 +352,9 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
     }
   }];
 
-  return stickyHeader ?: [super hitTest:point withEvent:event];
+  CGPoint convertedPoint = [stickyHeader convertPoint:point fromView:self];
+  return stickyHeader ? [stickyHeader hitTest:convertedPoint withEvent:event] :
+                        [super hitTest:point withEvent:event];
 }
 
```

Comment:
Could change `__block UIView *stickyHeader` to `__block UIView *hitView` and set it inside the enumeration loop, avoiding an extra hitTest: call at the end of this method.